content/posts/scientific-python/dev-summit-2/index.md

```diff
@@ -0,0 +1,97 @@
+
+### SPECs
+
+The Scientific Python Ecosystem Coordination documents (SPECs) aim to improve coordination of technical development across the ecosystem.
+
+Several new SPECs were started:
+
+- [SPEC-?: Dispatching (`spatch`)](https://hackmd.io/yI1iAqekQIq0a4jLS9WPyw)
+- [SPEC-8: Securing The Release Process](https://scientific-python.org/specs/spec-0008/)
+- [SPEC-9: Governance](https://github.com/scientific-python/specs/pull/323)
+- [SPEC-10: Changelog and release documentation](https://github.com/scientific-python/specs/pull/321)
+- [SPEC-12: Formatting mathematical expressions](https://github.com/scientific-python/specs/pull/326)
+- [SPEC-13: Naming conventions](https://github.com/scientific-python/specs/pull/324)
+
+Some existing SPECs were discussed and improved:
+
+- [SPEC-7: Seeding pseudo-random number generation (SPRaNG)](https://scientific-python.org/specs/spec-0007/)
+
+### Tooling
+
+- We created a new tools team to handle the ever-growing [list of tools we maintain](https://tools.scientific-python.org/).
+- Eric added his [circleci-artifacts-redirector-action](https://github.com/scientific-python/circleci-artifacts-redirector-action) to the suite.
+- Matthias brought over his [backport bot](https://github.com/scientific-python/MeeseeksDev) and set up a maintenance team.
+
+### SciPy
+
+Several of the SciPy developers were present, and we used the opportunity to celebrate Dan Schult joining as a core developer.
+Matt and Pamphile did some work on the new distribution infrastructure, Dan worked on sparse (remotely with CJ), and newly-supported [`const` statements were added to Cython code](https://github.com/scipy/scipy/pull/20891) got reviewed and merged.
+
+### Unplanned collaborations
+
+As is the nature of these events, some collaborations arise spontaneously.
+E.g., Nick and Ariel explored using Awkward Array for neuro-tractography, while Nick and Mridul explored using [scipp](https://scipp.github.io/index.html) for high-energy physics data.
+Guen worked on telemetry, while Inessa and Sanket discussed community surveys and governance.
+Sebastian and Thomas [discussed parallelization APIs](https://hackmd.io/84thx0ucQ2ab17ZYrBhWRw).
+
+### Conclusion
+
+Numerous other PRs were made, of which a number were probably not even captured in the [worklog](https://hackmd.io/wsJVTMYdQGG_Zgz7rgxSzw).
+But, besides the inherent satisfaction of working together with this great group, the best feature of the summit was that we were able to hang out, bonding over our communal joys and struggles—both technical and personal.
+
+We are grateful to the ecosystem developers who gave up their time to attend the summit (many had to put in leave _just to do more work_!).
+The summits are valuable, and translate to a lot of work work getting done and decisions being made.
```

Comment:
Unless this was here for emphasis:
```suggestion
The summits are valuable, and translate to a lot of work getting done and decisions being made.
```

Reply:
Added in latest commit, and included you as author.